yarn.lock

```diff
@@ -3656,7 +3661,7 @@ lodash.merge@^4.6.2:
 
 lodash@^4.17.20, lodash@^4.17.21:
   version "4.17.21"
-  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.21.tgz#679591c564c3bffaae8454cf0b3df370c3d6911c"
+  resolved "https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz"
```

Comment:
I think some of these changes are using npm vs yarn. Use yarn for this package.

Reply:
deleted node_modules and installed via yarn install, should be good now